README.md

```diff
@@ -59,6 +52,25 @@ Example:
 resolution -mc eth,btc,DODGE,unknown -d brad.zil
 ```
 
+### JavaScript/NodeJs
+
+Install dependency
+```
+npm install @unstoppabledomains/resolution
+```
+
+Example:
+```
+const { Resolution } = require('@unstoppabledomains/resolution');
+
+const resolution = new Resolution();
+resolution.address('resolver.eth', "ETH")
+    .then(console.log)
+    .catch(console.error);
+```
+
+For more information take a look [documentation](https://docs.unstoppabledomains.com/#tag/resolution_library)
```

Comment:
I believe we want to remove Resolution Library section from the documentation, so we can just move all the Getting Started contents from there to this readme.

Reply:
any suggestions, where to redirect?